ABApplicationCore.js

```diff
@@ -278,6 +296,10 @@ module.exports = class ABApplicationCore extends ABMLClass {
 
       this.json.name = this.name;
 
+      this.json.versionNumber = this.versionData.versionNumber;
```

Comment:
we seem to be specifically storing the `.versionNumber` in `this.json` but we don't seem to be reading it out or accessing it as `this.json.versionNumber` anywhere.  

I think it would be just as good to just reference it as `this.versionData.versionNumber`.

So we probably don't need to store it like this.